esp-hal/src/gpio/interconnect.rs

```diff
@@ -1,4 +1,21 @@
 //! Peripheral signal interconnect using IOMUX or GPIOMUX.
+//! 
+//! ## Examples
```

Comment:
Just remove this, we're placing this example back and forth between the two modules.